flux-refineck/src/type_env.rs

```diff
@@ -385,8 +395,8 @@ impl BasicBlockEnvShape {
             BaseTy::Slice(ty) => BaseTy::Slice(Self::pack_ty(scope, ty)),
             BaseTy::Ref(r, ty, mutbl) => BaseTy::Ref(*r, Self::pack_ty(scope, ty), *mutbl),
             BaseTy::Array(ty, c) => BaseTy::Array(Self::pack_ty(scope, ty), c.clone()),
-            BaseTy::Alias(..) => tracked_span_bug!("packing of `AliasTy` is not implemented"),
-            BaseTy::Int(_)
+            BaseTy::Alias(..) // => tracked_span_bug!("packing of `AliasTy` is not implemented"),
```

Comment:
I'm not entirely sure what's the right thing to do here, but we should at least add an `assert` for `BaseTy::Alias` ensuring it doesn't have free variables, i.e., `assert!(scope.has_free_vars(bty))`.